CHANGELOG.md

```diff
@@ -4,6 +4,30 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.12.6](https://github.com/go-gitea/gitea/releases/tag/v1.12.6) - 2020-11-11
+
+* SECURITY
+  * Disallow urlencoded new lines in git protocol paths if there is a port (#13521) (#13525)
+* BUGFIXES
+  * Prevent panic on git blame by limiting lines to 4096 bytes at most (#13470) (#13492)
```

Comment:
```suggestion
  * Fix before and since query arguments at API (#13559) (#13560)
  * Prevent panic on git blame by limiting lines to 4096 bytes at most (#13470) (#13492)
```